README.md

```diff
@@ -98,5 +98,12 @@ If you'd like to refresh ACS data, there are a few steps you'll need to take:
 
 The `import_data` management command will update data if it already exists, so you shouldn't experience data loss during import. Still, we recommend that you practice caution and make a backup.
 
+## Permissioning
+
+This app employs two classes of users:
+
+1. `Staff users` have full access. They can run surveys as well as create, edit, and delete Users, Agencies, Studies, Study Areas, Surveys, and Locations. They can publish surveys so they can be run. Admins can also view all collected data and design data visualizations on the collected data pages. **To create a new staff user, both "Staff status" and "Superuser status" should be selected on the second page of the Django admin user creation interface.**
+2. `Field users` can run published surveys.
```

Comment:
Your comment about password management was a good one:

> One thing worth noting here is that field users have no way to change their own passwords—staff users will have to change it for them.

It'd be good to add a quick note here explaining this.